astropy/io/ascii/basic.py

```diff
@@ -16,29 +16,32 @@
 from . import core
 
 class BasicHeader(core.BaseHeader):
-    '''Basic table Header Reader
+    """
```

Comment:
I made updates according to other files in directory. I think - this will be better to write uniforms code, even in one directory or functionality.